db/db.go

```diff
@@ -102,8 +102,8 @@ func ShowTables() []string {
 func ExecuteChangeCommand(command string, errorMessage string) bool {
 	_, err := DB.Exec(command)
 	if err != nil {
-		log.Panicf("Failed to execute change command: %s", err)
-		return false
+		log.Println(errorMessage)
+		log.Fatalf("Failed to execute change command: %s", err)
```

Comment:
Can we merge these into 1 log.Fatalf("Failed to execute change command %s: %s", errorMessage, err)?